src/app/content/highlights/components/Highlights.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { useSelector } from 'react-redux';
+import myHighlightsEmptyImage from '../../../../assets/MHpage-empty-logged-in.png';
+import Loader from '../../../components/Loader';
+import { stripHtmlAndTrim } from '../../hooks/receiveContent';
+import { book as bookSelector } from '../../selectors';
+import { ArchiveTree, ArchiveTreeSection } from '../../types';
+import { stripIdVersion } from '../../utils/idUtils';
+import { summaryHighlights, summaryIsLoading } from '../selectors';
+import { SummaryHighlights } from '../types';
+import * as HStyled from './HighlightStyles';
+import * as Styled from './ShowMyHighlightsStyles';
+
+// tslint:disable-next-line: variable-name
+const Highlights = () => {
+  const book = useSelector(bookSelector);
+  const highlights = useSelector(summaryHighlights);
+  const isLoading = useSelector(summaryIsLoading);
+
+  if (book && Object.keys(highlights).length > 0) {
+    return <Styled.Highlights isLoading={isLoading}>
+      {isLoading ? <Styled.LoaderWrapper><Loader/></Styled.LoaderWrapper> : null}
+      {highlights[book.tree.id]
+        && <SectionHighlights
+          section={book.tree}
+          highlights={highlights}
+        />}
+      {book.tree.contents.map((chapter) => (
```

Comment:
this won't accurately find chapters, you need to flatten the tree and filter with `archiveTreeUtils.archiveTreeSectionIsChapter`, different books have different table of contents formatting and they're not always the first children of the book.

its also possible to have highlights in pages that are not in a chapter